MAUI/Busy-Indicator/Getting-Started.md

```diff
@@ -8,23 +8,39 @@ documentation: ug
 keywords : .net maui busy indicator, maui busy indicator, .net maui activity indicator, maui activity indicator, .net maui loading indicator, maui loading indicator.
 ---
 
-# Getting Started with .NET MAUI Busy Indicator (SfBusyIndicator)
+# Getting Started with .NET MAUI Busy Indicator
 
-## Getting started with .NET MAUI Busy Indicator
+This section guides you through setting up and configuring a
```

Comment:
This section guides you through setting up and configuring a [Busy Indicator](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfBusyIndicator.html?tabs=tabid-1)
in your .NET MAUI application. Follow the steps below to add a basic Busy Indicator to your project.

Reply:
Yes, I have changed.